### PR TITLE
fixed a typo in `docs/Overview.md`

### PR DESCRIPTION
**Pull request by Haobo-Zhao**
**Summary**

This PR fixed a typo in file `docs/Overview.md`, where better options for word `use` in the changed place are  `we` and `users`, and `we` sounds better to me and thus the PR.

**Test Plan**

No test needed.

### CODE REVIEW

File: docs/Overview.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ ReactDOM.render(
 );
 ```
 
-Since the release of React 16.8, use can use [Hooks](https://reactjs.org/docs/hooks-intro.html) as a way to work with `EditorState` without using a class.
+Since the release of React 16.8, we can use [Hooks](https://reactjs.org/docs/hooks-intro.html) as a way to work with `EditorState` without using a class.
```
Comment (robbertbrak) [inline docs/Overview.md]: How about "you" instead of "we"?

Comment (Haobo-Zhao) [reply docs/Overview.md]: thank you for the response Robbert, `you` is a good option too, you can definitely open a PR and see who gets picked 😉 , to me `we` sounds better personally.

Comment (robbertbrak) [reply docs/Overview.md]: Looks like this was already addressed in #2089.

Comment (claudiopro) [reply docs/Overview.md]: Hi @Haobo-Zhao @robbertbrak, thanks for proposing a fix for this typo! I prioritized the earlier PR as I think it's more fair to accept a fix from a contributor who spotted and addressed it first.

**Comment by claudiopro** [comment]
Already addressed with #2089, but thanks for proposing a fix @Haobo-Zhao!

**Pull request closed without merging**